src/Apps/Auction/Routes/__tests__/ConfirmBid.test.tsx

```diff
@@ -394,8 +394,17 @@ describe("Routes/ConfirmBid", () => {
       await page.agreeToTerms()
       await page.submitForm()
 
-      expect(mockPostEvent).toHaveBeenCalledTimes(1)
-      expect(mockPostEvent).toHaveBeenCalledWith({
+      expect(mockPostEvent).toHaveBeenCalledTimes(2)
+      expect(mockPostEvent.mock.calls[0][0]).toEqual({
```

Comment:
small thing: I would recommend updating the doc string to reflect that we're tracking 2 events: bid and registration